HuskStandalone/HuskStandalone.py

```diff
@@ -126,7 +130,10 @@ def RenderArgument(self):
         # They mention using the `--disable-dummy-raster-product` husk flag.
 
         arguments.append("--make-output-path")
-        arguments.append("--disable-dummy-raster-product")
+        
+        if self.GetPluginInfoEntryWithDefault( "Version", "" ).startswith("20"):

```

Comment:
I would actually do it in reverse - assume it's the latest version unless a specific Houdini version older than 20 is used. As such, in this case we could check if it does not start with `"20"`.

Reply:
let me know what do you think.
https://github.com/BigRoy/HuskStandaloneSubmitter/pull/2/commits/253fadbe7c76b96fde4aa977efd21636d4d9609b